gen3/bin/kube-setup-gearbox-middleware.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Deploy the metdata service.
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+
+setup_gearbox_middle() {
+  if g3kubectl describe secret gearbox-middleware-g3auto > /dev/null 2>&1; then
+    gen3_log_info "gearbox-middleware-g3auto secret already configured"
+    return 0
+  fi
+  
+  if [[ ! -f "$secretsFolder/gearbox-middleware.env" || ! -f "$secretsFolder/base64Authz.txt" ]]; then
+    local secretsFolder="$(gen3_secrets_folder)/g3auto/gearbox-middleware"
+    # go ahead and rotate the password whenever we regen this file
+    local password="$(gen3 random)"
```

Comment:
the directory has never been created. Add this `mkdir -m 0700 -p "$secretsFolder"`